Wrappers/Python/cil/optimisation/algorithms/PDHG.py

```diff
@@ -334,16 +328,16 @@ def set_up(self, f, g, operator, tau=None, sigma=None, initial=None, **kwargs):
         theta : Relaxation parameter, Number, default 1.0
         """
         print("{} setting up".format(self.__class__.__name__, ))
-        
+
         # Triplet (f, g, K)
         self.f = f
         self.g = g
         self.operator = operator
 
         self.set_step_sizes(sigma=sigma, tau=tau)
 
-        if kwargs.get('check_convergence', True):
-            self.check_convergence()
+        if kwargs.get('check_convergence_criterion', True):
+            self.check_convergence_criterion()
```

Comment:
this renaming is the only functional change to PDHG.

Why is this in this PR? 

I would suggest this file should be removed from the PR, as in the [discussion](https://github.com/TomographicImaging/CIL/pull/1143#pullrequestreview-908166603).

Reply:
Removed from this PR, see #1256 .